resources/css-test-fonts.css

```diff
@@ -0,0 +1,419 @@
+@charset "utf-8";
+
+@font-face {
+    font-family: "CSS Full-Width Orientation Test";
+    src: url(css-test-fonts/adobe-fonts/CSSFWOrientationTest.otf);
+}
+
+@font-face {
+    font-family: "CSS Half-Width Orientation Test";
+    src: url(css-test-fonts/adobe-fonts/CSSHWOrientationTest.otf);
+}
+
+@font-face {
+    font-family: "Ahem";
+    src: url(css-test-fonts/ahem/AHEM____.TTF);
+}
+
+@font-face {
+    font-family: "Ahem!";
+    src: url(css-test-fonts/ahem-extra/AHEM_Ahem!.TTF);
+}
+
+@font-face {
+    font-family: "MissingItalicOblique";
+    src: url(css-test-fonts/ahem-extra/AHEM_MissingItalicOblique.TTF);
+}
+
+@font-face {
+    font-family: "MissingNormal";
+    src: url(css-test-fonts/ahem-extra/AHEM_MissingNormal.TTF);
+}
+
+@font-face {
+    font-family: "SmallCaps";
+    src: url(css-test-fonts/ahem-extra/AHEM_SmallCaps.TTF);
+}
+
+@font-face {
+    font-family: "White Space";
+    src: url(css-test-fonts/ahem-extra/AHEM_WhiteSpace.TTF);
+}
+
+/*
+
+These can't be globally enabled, because of tests checking that
+'sans-serif', Ahem falls back to Ahem, rather than using the generic
+sans-serif.
```

Comment:
No. The CSS testsuite has tests that 'sans-serif' isn't recognised as the generic and falls back to the next thing in the list, and has tests that 'sans-serif' matches an installed font with that name.